1/ToDoApp/src/App.vue

```diff
@@ -7,7 +7,7 @@
 </template>
 
 <script>
-  import ToDoList from "./components/ToDoList.vue";
+  import ToDoList from "./components/ToDoList/ToDoList.vue";
```

Comment:
That's how we do it at https://github.com/human-connection/human-connection as well. All tests, storybooks, source files go together.